first-bot/adding-a-config-file.md

```diff
@@ -31,9 +31,9 @@ Simply take the following example, and create a new file in the same folder as y
 At the top of your bot file, you need to add a line that will load this configuration, and put it in a variable. This is what it looks like:
 
 ```javascript
-const Discord = require("discord.js");
+const { Client, Intents } = require("discord.js");
```

Comment:
Again with consistency.

Reply:
I'll fix all the consistency issues up now.